software/samtools/merge/meta.yml

```diff
@@ -0,0 +1,42 @@
+name: samtools_merge
+description: Merge BAM file
+keywords:
+    - merge
+    - bam
```

Comment:
```suggestion
    - bam
    - sam
    - cram
 ```

Reply:
I thought of that, but currently we expect only bam as output

Reply:
but it should be good as input, probably, so I'll accept your suggestions